src/Catalyst.Core.Lib.Tests/UnitTests/IO/Events/TransactionReceivedEventTests.cs

```diff
@@ -66,7 +68,7 @@ public void Can_Send_Error_To_Invalid_Transaction()
         {
             _transactionValidator.ValidateTransaction(Arg.Any<PublicEntry>())
                .Returns(false);
-            _transactionReceivedEvent.OnTransactionReceived(new TransactionBroadcast {PublicEntry = new PublicEntry()}
+            _transactionReceivedEvent.OnTransactionReceived(new TransactionBroadcast {PublicEntry = new PublicEntry{SenderAddress = new byte[32].ToByteString(), Timestamp = Timestamp.FromDateTime(DateTime.UtcNow)}}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Split this 214 characters long line (which is greater than 200 authorized).](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=5051857)